composer.json

```diff
@@ -16,27 +15,28 @@
             "homepage": "http://github.com/KnpLabs/KnpTimeBundle/contributors"
         }
     ],
-
     "require": {
-        "php":                      ">=5.3.3",
-        "symfony/framework-bundle": "~2.0"
+        "php": ">=5.3.3",
+        "symfony/dependency-injection": "^2.7",
```

Comment:
I won't comment other changes.
BUT.
We should  allow users with Symfony <= 2.7 to use this bundle, shouldn't we?